packages/tracing/src/browser/browsertracing.ts

```diff
@@ -69,7 +69,7 @@ export interface BrowserTracingOptions extends RequestInstrumentationOptions {
    *
    * Default: undefined
    */
-  _metricOptions?: Partial<{ _reportAllChanges: boolean }>;
+  _metricOptions?: Partial<{ _reportAllChanges: boolean, _reportLongTasks: boolean }>;
```

Comment:
I didn't really want to make another option bundle here, re-using something called 'metric options' when it makes spans doesn't sound totally right to me though.

Reply:
Can we just turn it on by default? Do we really need to gate to an option?

Reply:
Fair.